server/events/terraform/terraform_client.go

```diff
@@ -105,7 +107,7 @@ func NewClientWithDefaultVersion(
 	tfDownloader Downloader,
 	usePluginCache bool,
 	fetchAsync bool,
-	terraformOutputChan chan<- *models.TerraformOutputLine,
+	terraformOutputChan chan<- *models.ProjectCmdOutputLine,
```

Comment:
update this to `projectCmdOutputHandler handlers.DefaultProjectCommandOutputHandler`

Reply:
```suggestion
	projectCmdOutputHandler handlers.DefaultProjectCommandOutputHandler,
```